testsupport/build.gradle

```diff
@@ -2,12 +2,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 android {
-    compileSdkVersion 29
+    compileSdkVersion compile_sdk_version
 
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 29
+        minSdkVersion min_sdk_version
```

Comment:
@AnasNaouchi is it OK that the `minSdkVersion` here is going from 16 → 21?

Reply:
Yeah should not be a problem